### PR TITLE
Change <img> tags to NextJS Images

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/4012

Resolves https://github.com/opencollective/opencollective/issues/4191

### CODE REVIEW

File: components/become-a-host/WhoAreFiscalHostsSection.js [hunk 1]
```diff
@@ -119,9 +119,11 @@ const WhoAreFiscalHosts = () => {
           width={['222.74px', null, null, null, '330px']}
           height={['142.48px']}
         >
-          <Illustration
+          <NextIllustration
             src="/static/images/become-a-host/whoAreFiscalHost-illustration.png"
             alt="Who are Fiscal Hosts illustration"
+            width={330}
```
Comment (Betree) [inline components/become-a-host/WhoAreFiscalHostsSection.js:125]: There is a change on this one

Peek 2021-05-07 09-04

File: components/become-a-sponsor/TransparencySection.js [hunk 2]
```diff
@@ -56,34 +56,30 @@ const Transparency = () => (
...
-      <Illustration
-        display={['none', 'none', 'block', null, 'none']}
-        src="/static/images/become-a-sponsor/transactionScreenshot-md.png"
-        alt="Transaction Screenshot"
-      />
-      <Illustration
-        display={['none', 'none', 'none', null, 'block']}
-        src="/static/images/become-a-sponsor/transactionScreenshot-lg.png"
+      <NextIllustration
+        src="/static/images/become-a-sponsor/transactionScreenshot.png"
```
Comment (Betree) [inline components/become-a-sponsor/TransparencySection.js:60]: On Firefox I end up with scrollbars there

Peek 2021-05-07 09-07

File: components/become-a-host/WhoAreFiscalHostsSection.js (hunk 1 above)
Comment (SudharakaP) [reply components/become-a-host/WhoAreFiscalHostsSection.js]: Thanks. Done. 👍🏽

**Comment by SudharakaP** [comment]
@Betree : Thanks for the review. I've corrected the issues. Let me know if you see anything else. 😄

**Review by Betree — APPROVED** [review]
I haven't re-tested but the rest looked good :+1:

**Pull request merged**